recipes/llnl-sina/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "llnl-sina" %}
+{% set version = "1.9.1" %}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    url: https://github.com/LLNL/Sina/archive/{{ version }}.tar.gz
+    sha256: 28ca48c89b8e8280b41458f8a8eabf51408c68bc3a45a433258eb548b5447e8e
+
+build:
+  number: 0
+  noarch: python
+
```

Comment:
Please remove the extra lines, the build scripts, and try something like:

```yaml
script:
  - cd python
  - {{ PYTHON }} -m pip install . -vv
```

here in build. That way we can have everything in a single file.